examples/networkpolicy/kubernetes-azure-hybrid.json

```diff
@@ -0,0 +1,52 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {
+        "networkPolicy": "azure"
+      }
+    },
+    "masterProfile": {
+      "count": 1,
+      "dnsPrefix": "",
+      "vmSize": "Standard_D2_v2"
+    },
+    "agentPoolProfiles": [
+      {
+        "name": "linuxpool1",
+        "count": 2,
+        "vmSize": "Standard_D2_v2",
+        "availabilityProfile": "AvailabilitySet",
+        "ipAddressCount": 20,
```

Comment:
As discussed, I am using a single large /12 subnet for all masters, agents and pods when VNET integration is enabled. This allows us to allocate 128 IP addresses per node, more than the default --max-pods value of 110, fixing the root cause. This also allows creation of huge Kubernetes clusters with 1200 nodes now and more in the future.